### ISSUE TITLE
FF ESR doesn't redirect to html.duckduckgo.com when NoScript is blocking JS

### ISSUE THREAD

**Issue by hackerncoder** [hackademix/noscript#192]
When I have NoScript installed and set to blocking JS, whenever I search anything on duckduckgo.com it doesn't redirect me to html.duckduckgo.com. But only if I use Firefox ESR (or derivatives like Tor Browser), I tested it with stable and nightly which both worked. I'm on debian sid and buster.

**Comment by hackademix** [comment]
Please double check that the DEFAULT preset has the "noscript" capability checked (enabled): without it, the <NOSCRIPT> tags don't get emulated and therefore the <META> refresh tags which produce the redirect don't work.

**Comment by hackerncoder** [comment]
It does. TB always adds noscript.

Edit: FF ESR also has it enabled.

**Comment by hackademix** [comment]
Cannot reproduce on Fx 78.10.1, NoScript 11.2.7.
Could you please share with me your NoScript Options>Export config file?
Thanks!

**Comment by hackerncoder** [comment]
```
{
  "policy": {
    "DEFAULT": {
      "capabilities": [
        "frame",
        "fetch",
        "other",
        "noscript"
      ],
      "temp": false
    },
    "TRUSTED": {
      "capabilities": [
        "script",
        "object",
        "media",
        "frame",
        "font",
        "webgl",
        "fetch",
        "ping",
        "noscript"
        "unchecked_css",
        "other"
      ],
      "temp": false
    },
    "UNTRUSTED": {
      "capabilities": [],
      "temp": false
    },
    "sites": {
      "trusted": [
        "§:addons.mozilla.org",
        "§:afx.ms",
        "§:ajax.aspnetcdn.com",
        "§:ajax.googleapis.com",
        "§:bootstrapcdn.com",
        "§:code.jquery.com",
        "§:firstdata.com",
        "§:firstdata.lv",
        "§:gfx.ms",
        "§:google.com",
        "§:googlevideo.com",
        "§:gstatic.com",
        "§:hotmail.com",
        "§:live.com",
        "§:live.net",
        "§:maps.googleapis.com",
        "§:mozilla.net",
        "§:netflix.com",
        "§:nflxext.com",
        "§:nflximg.com",
        "§:nflxvideo.net",
        "§:noscript.net",
        "§:outlook.com",
        "§:passport.com",
        "§:passport.net",
        "§:passportimages.com",
        "§:paypal.com",
        "§:paypalobjects.com",
        "§:securecode.com",
        "§:securesuite.net",
        "§:sfx.ms",
        "§:tinymce.cachefly.net",
        "§:wlxrs.com",
        "§:yahoo.com",
        "§:yahooapis.com",
        "§:yimg.com",
        "§:youtube.com",
        "§:ytimg.com",
      ],
      "untrusted": [],
      "custom": {}
    },
    "enforced": true,
    "autoAllowTop": false
  },
  "local": {
    "debug": false,
    "showCtxMenuItem": true,
    "showCountBadge": true,
    "showFullAddresses": false,
    "amnesticUpdates": false,
    "storage": "local",
    "uuid": "3f59ffbd-d8c1-4841-954e-77536a502d1b",
  },
  "sync": {
    "global": false,
    "xss": true,
    "cascadeRestrictions": true,
    "overrideTorBrowserPolicy": false,
    "clearclick": true,
    "storage": "sync"
  },
  "xssUserChoices": {}
}
```

**Comment by hackerncoder** [comment]
On my Windows PC (where I have never had FF ESR) I installed ESR and then NoScript, didn't change anything, and a search on DDG had the same problem.

**Comment by hackademix** [comment]
OK, thanks, managed to reproduce. A fix is on its way.

**Comment by hackademix** [comment]
Fixed in 11.2.8 (current public stable).
